libs/algorithms.py

```diff
@@ -2,17 +2,39 @@
 # Module to store various algorithms
 #
 # Author: XCC
-# Date: 10/25/2020
-#
+# Date: 11/03/2020
+# Edited by: Ayaan D.
 
 
-def bubbleSort(list):
-    return list
+def bubbleSort(ls):
+    lengthOfList = len(numList)
+    for i in range(lengthOfList):
+        for j in range(0, lengthOfList - i - 1):
+            if numList[j] > numList[j+1]:
+                numList[j], numList[j+1] = numList[j+1], numList[j]
```

Comment:
oh i forgot i could do that thingy on line 14

Reply:
yeah i completely forgot about it as well. i learned sorting algos in java so doing them in python took me a lot longer than it should've lol